packages/docker/data_stream/image/manifest.yml

```
@@ -0,0 +1,34 @@
+type: metrics
+title: Docker image metrics
+release: beta
+streams:
+  - input: docker/metrics
+    enabled: false
+    vars:
+      - name: hosts
+        type: text
+        title: Hosts
+        multi: true
+        required: true
+        show_user: true
+        default:
+          - unix:///var/run/docker.sock
+      - name: period
+        type: text
+        title: Period
+        multi: false
+        required: true
+        show_user: true
+        default: 10s
+      - name: labels.dedot
+        type: bool
+        title: De-Dot labels
+        multi: false
+        required: false
+        show_user: true
+        default: true
```

Consider to disable dedotting if we start using `flattened` type for labels.
```suggestion
        default: false
```